ledger/src/ui/src/onboard.h

```diff
@@ -29,18 +29,13 @@
 #include "os_io_seproxyhal.h"
 #include "pin.h"
 
-// passphrase will be prefixed with "MNEMONIC" from BIP39, the passphrase
-// content shall start @ 8
-unsigned int bolos_ux_mnemonic_from_data(unsigned char *in,
-                                         unsigned int inLength,
-                                         unsigned char *out,
-                                         unsigned int outLength);
-
 // Onboard context
 typedef struct {
-    char client_seed[MAX(
-        32, sizeof(bagl_icon_details_t) + BOLOS_APP_ICON_SIZE_B - 1)];
-    char words_buffer[257];
+    union {
+        unsigned char words_buffer[257];
```

Comment:
Is there a constant available we can use instead of plain `257`? Maybe within the `bolos_ux_mnemonic_from_data` file there is or we could add one instead?